PRs open buddy

Contrast-AlexB opened the pull request.

Contrast-AlexB committed:
PRs open buddy

prodion23 commented:
Hi Contrast-AlexB!

Thanks for putting in a PR, I don't think we should copy over the example file to config.yaml because if a config.yaml is present then it will be overwritten(we will make sure to specify that a config.yml is expected in the Readme)

astone123 committed:
Modify copy statement and add expose instruction

astone123 commented:
I added a command to expose port 8000 and modified the copy command to just copy whatever they have as `config.yaml` into the container. I'll write up a readme that explains how to run this with the Dockerfile.

Contrast-AlexB commented:
> I added a command to expose port 8000 and modified the copy command to just copy whatever they have as `config.yaml` into the container. I'll write up a readme that explains how to run this with the Dockerfile.

Hi astone123, 

I appreciate your eagerness but please refrain from hijacking other people's PRs. This could scare away any potential contributors.

This project looks great, can't wait to sell it to Amazon
